app/src/main/java/net/squanchy/navigation/HomeActivity.java

```diff
@@ -56,8 +57,12 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
     protected void onStart() {
         super.onStart();
         selectInitialPage(currentSection);
+        proximityService = HomeInjector.obtain(this).service();
+        proximityService.startRadar();
```

Comment:
Can you merge `develop` in the branch and put these two lines behind the [feature flag](https://github.com/rock3r/squanchy/blob/develop/app/src/main/java/net/squanchy/navigation/HomeActivity.java#L71-L73)?